purchase_discount/report/purchase_report.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+# © 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# © Odoo S.A. (https://www.odoo.com/)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models, tools
+
+
+class PurchaseReport(models.Model):
+    _inherit = "purchase.report"
+
+    @api.model_cr
+    def init(self):
+        tools.drop_view_if_exists(self._cr, 'purchase_report')
+        self._cr.execute("""
```

Comment:
This is also breaking inheritance. Do something similar to what I have done in https://github.com/OCA/purchase-reporting/blob/9.0/purchase_reporting_weight/reports/purchase_report.py